scanners/opensecurity/mobsfscan/rules.yaml

```diff
@@ -371,6 +371,19 @@ rules:
       and retrieves the contents of this URL, but it does not sufficiently ensure
       that the request is being sent to the expected destination.
     ref: https://github.com/returntocorp/semgrep-rules/
+  CWE-919:
+    categories:
+    - ALL
+    - cwe-top-25
+    - cwe-919
+    - boost-baseline
+    - boost-hardened
+    - owasp-top-10
+    group: top10-weakness-in-mobile-applications
```

Comment:
Ajin, the list of groups is curated the group you use here doesn't exist in our list of "groups". 
What we have for now:
  top10-broken-access-control:
  top10-crypto-failures:
  top10-injection:
  top10-insecure-design:
  top10-security-misconfiguration:
  top10-vulnerable-components:
  top10-id-authn-failures:
  top10-software-data-integrity-failures:
  top10-security-logging-monitoring-failures:
  top10-server-side-request-forgery:
  stored-secrets:
  cloud-weak-configuration:
  cloud-resources-public-access:
  cloud-insecure-iam:
  cloud-unencrypted-resources:
  cloud-weak-secrets-management:
  supply-chain-scm-weak-configuration:
  supply-chain-cicd-vulnerable-pipeline:
  supply-chain-cicd-weak-configuration:
  supply-chain-missing-artifact-integrity-verification:
  supply-chain-malicious-dependency:
  supply-chain-unrestrained-use-of-3rd-party-services:
  
So I suggest for now we could either use :
top10-vulnerable-components:
or
top10-insecure-design

Reply:
mobsfscan has a couple of rules under CWE-919 https://github.com/search?q=repo%3AMobSF%2Fmobsfscan%20cwe-919&type=code
These are general weakness in mobile application code that doesn't come under any specific CWEs. For now, I will choose `top10-insecure-design` as the group. In future, it would be nice to have a group for mobile weaknesses.